tests/contrib/funsor/test_vectorized_markov.py

```diff
@@ -0,0 +1,468 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import torch
+
+from torch.distributions import constraints
+
+from pyro.ops.indexing import Vindex
+
+# put all funsor-related imports here, so test collection works without funsor
+try:
+    import funsor
+    import pyro.contrib.funsor
+    from pyroapi import distributions as dist
+    funsor.set_backend("torch")
+    from pyroapi import handlers, pyro, pyro_backend
+except ImportError:
+    pytestmark = pytest.mark.skip(reason="funsor is not installed")
+
+
+def model_0(data, history, vectorized):
+    x_dim = 3
+    pyro.set_rng_seed(0)
+    pyro.get_param_store().clear()
```

Comment:
nit: remove these two lines from all of the models, they are handled in our pytest setup